Recruit-Training/Advanced-Recruit-Training/Embedded-Teaser/Training/Core/Src/Training.c

```diff
@@ -2,5 +2,119 @@
 
 void trainingTask(uint8_t* data)
 {
+    loadMotorData(data[0], data[1]);
+    loadLightData(data[2]);
 
+    outputArray[0] = data[0];
+    outputArray[1] = data[1];
+    outputArray[2] = data[2];
+    validData = 0b00000111;
+
+    if(!checkMotors())
+    {
+        validData = 0b100 & validData;
+        outputArray[0] = 0b11111111;
+        outputArray[1] = 0b11111111;
```

Comment:
Apologies if the instruction was unclear but you should only write to `outputArray` if `data` is valid so you shouldn't be assigning `outputArray[0]` and `outputArray[1]` to `0b11111111`

Reply:
Once you change this, it's probably going to break your tests in `TrainingTest.c` because you were mostly checking against `0b11111111`